src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CreateCaseCallbackService.java

```diff
@@ -100,30 +109,50 @@ public CreateCaseCallbackService(
         ExceptionRecord exceptionRecord,
         ServiceConfigItem configItem,
         long caseId,
-        boolean ignoreWarnings
+        boolean ignoreWarnings,
+        String idamToken,
+        String userId
     ) {
+        String caseIdStringify = Long.toString(caseId);
+
+        log.info("Start creating exception record for {} {}", configItem.getService(), caseIdStringify);
```

Comment:
Worth adding `service` and `case ID` into log message. I can see it was already like this, not sure why you changed it.

Reply:
Perhaps because code was done long since, just rebased. Sorry about that. Can add now but will review logging in future PR